panel/src/helpers/string.js

```diff
@@ -80,7 +80,7 @@ export function stripHTML(string) {
 export function template(string, values = {}) {
   const resolve = function (parts, data = {}) {
     const part  = escapeHTML(parts.shift());
-    const value = data[part] || null;
+    const value = data[part] !== undefined ? data[part] : null;
```

Comment:
A simpler solution: `data[part] ?? null`. Vite will transform optional chaining for us.

Reply:
Yes you are right, when I used null coalescing operator before, npm run was giving an error when doing build. No error now. I think it's starting to fully support it.

Reply:
Maybe prior to Vite? Vite will run the code through esbuild, which supports the nullish coalescing operator.